caim_base/templates/awg/manage/applications/list.html

```diff
@@ -98,23 +98,36 @@ <h5 class="text-reset">{{ app.fosterer.firstname }} {{ app.fosterer.lastname }}<
               Pets: No
             </div>
             {% endif %}
-          </div>
-          <p class="flex-grow-0 py-0 px-3 m-0 text-uppercase fs-6 text-light-emphasis">Animal Requested </p>
-          <div class="card-body d-flex justify-content-between flex-grow-0">
-              <div class="d-flex align-items-center">
-                <img
-                  style="background-color: #eee; clip-path: circle(); width: 30px; height: 30px;"
-                  src="{{ app.animal.primary_photo.url|image_resize:'75x75 crop' }}"
-                >
-                <span class="px-1">{{ app.animal }}</span>
+            <p class="flex-grow-0 mt-3 py-0 px-3 m-0 text-uppercase fs-6 text-light-emphasis">Animal Requested </p>
```

Comment:
I moved this into the card-body div. Before it was just between the card-body and the card-footer. Added `mt-3` to compensate the vertical spacing